src/Assets/Images/image.js

```diff
@@ -5,6 +5,12 @@ const Images = {
   mixedNuts: require("../Images/mixed nuts.jpg").default,
```

Comment:
<img width="1792" alt="" src="https://user-images.githubusercontent.com/10892977/161047044-87596a82-d2be-45f0-9116-989688232797.png">

UI suggestions:
- provide a document tiltle and favicon for your website
- the search bar can be more stretched out. make it  dynamic width (with a ceiling) so it can take up the empty space
- try to keep the footer at the bottom of the screen even when there is less content

Reply:
Thanks rohit for the UI suggestions i have noted them down will implement them after all the P0s are done